web/crux/templates/link-ace.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "LinkAce",
+  "description": "LinkAce is a self-hosted archive to collect links of your favorite websites.\nREQUIREMENTS: After deployment, exec in the linkace-app container and add 'chmod 777' privileges to the \\app\\.env file.",
+  "registries": [
+    {
+      "name": "Docker Hub Library",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "library"
+      }
+    },
+    {
+      "name": "Docker Hub Linkace",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "linkace"
+      }
+    }
+  ],
+  "images": [
+    {
+      "name": "linkace-db",
+      "registryName": "Docker Hub Library",
+      "image": "mariadb",
+      "tag": "10.7",
+      "config": {
+        "name": "linkace-db",
+        "volumes": [
+          {
+            "name": "db",
+            "path": "/var/lib/mysql",
+            "type": "rw"
+          }
+        ],
+        "environment": [
+          {
+            "key": "MYSQL_ROOT_PASSWORD",
+            "value": "someDbPassword"
+          },
+          {
+            "key": "MYSQL_USER",
+            "value": "user"
+          },
+          {
+            "key": "MYSQL_DATABASE",
+            "value": "linkacedb"
+          }
+        ],
+        "secrets": [
+          {
+            "key": "MYSQL_PASSWORD",
+            "required": true
+          }
+        ],
+        "networks": ["linkace-network"],
+        "networkMode": "bridge",
+        "restartPolicy": "unless_stopped",
+        "deploymentStrategy": "recreate"
+      }
+    },
+    {
+      "name": "linkace-app",
+      "registryName": "Docker Hub Linkace",
+      "image": "linkace",
+      "tag": "simple",
+      "user": 0,
+      "config": {
+        "expose": "expose",
+        "ingress": {
+          "host": "localhost",
+          "name": "linkace"
+        },
+        "name": "linkace-app",
+        "ports": [
+          {
+            "external": 6780,
+            "internal": 80
+          }
+        ],
+        "environment": [
+          {
```

Comment:
The app key is an important secret used to encrypt/decrypt data in LinkAce. It should probably added as a secret then.

Reply:
It's just a template (example), users have to change it when they use their own instance.

Reply:
But yea, a good point we'll move this key to the secret.

Reply:
Thanks for your review, APP_KEY is a secret from now. 👍

Reply:
Thanks for taking the time to create the template!